### DISCUSSION TITLE
Typescript Migra Help & Discussions

### DISCUSSION THREAD

**Discussion by amerryma** [mmkal/slonik-tools#425] [General]
Hi, I just stumbled upon your implementation of Migra in Typescript. This is actually something I've been wanting to do for a long time now. I cannot believe it's been basically done already!

I have a lot of ideas around next steps here, and I've been managing a fork of Migra to manage more complex diff cases and later versions of Postgres. I'd love to have a discussion around the future of pgkit and how I could help it's development.

**Comment by mmkal** [comment]
Nice! If you're on discord, DM me there (also mmkal). Or Twitter, linked in my GitHub profile.

**Reply by amerryma** [reply]
Ok just did, I'm 'uber'
